### ISSUE TITLE
Require 5/5 registered players when playing a season match

### ISSUE THREAD

**Issue by superjova** [chairleague/hots-rules#3]
Change 4/5 to 5/5 checked-in players from the roster. Matches cannot start until 5/5 from both teams are registered.

**Comment by Durgulach** [comment]
We have an 8 man roster that uses all 8 plauers due to job/time constraints, so far we have inly had one night out if about 7 where we almost had to use a sub.

I would say 5/5 is workable but I also know that if I were the opponent I would rather olay the match with a sub then get a forfit win that week amd olay TL.  At the same time, something variational like leaving it as optional for the fully present team whether a sub is ok for the match seems likely to breed resentment.

**Comment by Ronin89** [comment]
I agree with Durg...if the opposing team agrees then it should be allowed

**Comment by johnafogarty4** [comment]
if the system is set to require 5/5 whether the team agrees or not, the game won't be playable.  I actually support this change, the challenge is what happens if the roster is locked to 8, and 4 people get called into work that night, you have to forfeit and no one likes that.  

I don't think this rule is actually do-able in the spirit of CL.  However I do think that this rule may work at the paid tier, where teams have some skin in the game, and are a little more serious than the more casual CL divisions.

**Comment by superjova** [comment]
If this is a nice to have, that's ok too. But we'll need a separate issue created for sub rules

**Comment by superjova** [comment]
This rules has stayed as 4 players required to check-in
